lib/mayaUsd/ufe/UsdUIInfoHandler.cpp

```diff
@@ -68,6 +68,24 @@ void addMetadataCount(
         addMetadataStrings(refs.size(), tooltip, needComma, singular, plural);
     }
 }
+
+std::vector<double> getInvisibleColor()
+{
+    static std::vector<double> rgb;
+    static bool                initialized = false;
+    if (!initialized) {
+        MDoubleArray outlinerInvisibleColor;
+        if (MGlobal::executeCommand(
+                "displayRGBColor -q \"outlinerInvisibleColor\"", outlinerInvisibleColor)
```

Comment:
Why is this command being echo'd in the Script Editor? The default value is `displayEnabled=False` so it should not echo. Are you saying that if echo all commands is on in the script editor then this will be echo'd even though we used false for display?

Reply:
If that is true then we need to find another way to get the color that doesn't involve executeCommand.

Reply:
IDK why it gets echoed even though echoing is off. Probably "echo all commands" overrides the flag?